mig-agent/config.go

```diff
@@ -167,6 +168,9 @@ type globals struct {
 
 	// Add the private client key below.
 	agentKey []byte
+
+	// Lets you target specific agents
```

Comment:
Small comment, can be used beyond just targeting (e.g., the tags just generally describe some properties of the agent, but can be used for targeting too).